src/fft/polynomial.rs

```diff
@@ -91,16 +90,22 @@ impl Polynomial {
         result
     }
 
-    /// Returns the degree of the [`Polynomial`].
+    /// Returns the degree, i.e. the index of the highest non-zero coefficient,
```

Comment:
I understand what you say, but could it seem that you are taking the highest coefficient? Maybe say the highest index among all coeffs...?